src/tests/reducers/profileReducer.test.js

```diff
@@ -0,0 +1,81 @@
+import profileReducer from '../../store/reducers/profile_reducer';
+import * as types from '../../store/actions/actionTypes';
```

Comment:
@BrianSerem, are you using everything from `../../store/actions/actionTypes`, if not, only import what you will need.

Reply:
I have removed this and import only what I need.